scripts/ccvi/parse_ccvi.ts

```diff
@@ -38,6 +38,9 @@ async function main() {
   const metroRows = await readCsv('ccvi_metros.csv');
 
   const rows = [...stateRows, ...countyRows, ...metroRows];
+  const numberize = (x: any) => {
+    return Number(toNumber(x).toString().substring(0, 4));
```

Comment:
Cuts stringified float number to 4 characters, e.g., "0.21341451345" => "0.21"

We can do a more rigorous rounding / precision reduction if it's warranted, this was a quick-and-dirty way to see if it would be an improvement.

At the very least would probably want to add a comment. explaining what we're doing.

Reply:
How about `Number(x.toFixed(2))` ?